### DISCUSSION TITLE
Can current-day highlight be changed/removed?

### DISCUSSION THREAD

**Discussion by CamJak** [vkurko/calendar#98] [Q&A] [answered]
I am wondering if the highlight that happens to show which day is current could be modified and/or removed and if so how? Really like the component but cannot figure this out.

**Comment by vkurko** [comment, marked as answer]
You can override the css classes that the calendar uses with [theme](https://github.com/vkurko/calendar#theme) setting. For the current day, `theme.today` is used. You can override it with an empty string.

**Reply by CamJak** [reply]
That's what we needed, thanks!
